index.js

```diff
@@ -57,34 +54,28 @@ async function run() {
 		}
 
 		// Generate markdown content
-		const releaseBody = [];
-
-		if (header) {
-			releaseBody.push(header + '\n');
-		}
-
+		const commitEntries = [];
 		if (commits.length === 0) {
-			releaseBody.push('_Maintenance release_');
+			commitEntries.push('_Maintenance release_');
 		} else {
 			for (const {hash, title} of commits) {
-				releaseBody.push(`- ${hash} ${title}`);
+				const line = commitTemplate
+					.replace('{hash}', hash)
+					.replace('{title}', title)
+					.replace('{url}', repoURL + '/commit/' + hash);
+				commitEntries.push(line);
 			}
 		}
 
-		if (footer) {
-			releaseBody.push('\n' + footer);
-		}
-
-		if (includeRange) {
-			releaseBody.push(`\n[\`${range}\`](https://github.com/${owner}/${repo}/compare/${range})`);
-		}
-
 		const octokit = getOctokit(core.getInput('token'));
 		const createReleaseResponse = await octokit.repos.createRelease({
 			repo,
 			owner,
 			tag_name: pushedTag, // eslint-disable-line camelcase
-			body: releaseBody.join('\n'),
+			body: releaseTemplate
+				.replace('{version}', pushedTag)
```

Comment:
This is already the title of the tag, I don't think it's needed.

Reply:
It is only replaced if it exists, probably for people who want to generate `changelog.md` from releases.

Also thinking about it now, feels dumb. Will remove this.